process/block/displayBlock.go

```diff
@@ -304,6 +346,23 @@ func (txc *transactionCounter) displayTxBlockBody(
 	return lines
 }
 
+func getShardName(shardID uint32) string {
+	var shardStr string
+
+	switch shardID {
+	case core.MetachainShardId:
+		shardStr = "MetaChain"
+	case core.MainChainShardId:
+		shardStr = "MainChain"
+	case core.SovereignChainShardId:
```

Comment:
Just for this particular display func, default case here is to fmt.Sprintf(shardID).
But agree with your suggestion. Will add a task in Jira for this.
This approach however should be a bit tricky on node, since we will not use a const anymore, but rather a const +offset from this registration I guess?

Reply:
That new value will come from the config - the SDK. There will be a factory SC on mainnet which will increase this ID when registering the new sovereign.